{"name": "test_error_app_key", "status": "passed", "description": "错误的app_key参数", "start": 1599987460672, "stop": 1599987460702, "uuid": "9b9e069a-791b-4784-b47e-5e2ae55d0396", "historyId": "456ba0e2a8e490d60babd01b2b6bb235", "testCaseId": "b200633efbe84b70d71eaefc1cc76b1d", "fullName": "case.test_user_login.Login_user#test_error_app_key", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "Login_user"}, {"name": "host", "value": "V_LFJJLIU-NB0"}, {"name": "thread", "value": "22832-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_user_login"}]}